sirius-pipeline/README.md

```diff
@@ -1,2 +1,43 @@
-# Sirius Pipeline P4 Behavior Models
-**TODO**
\ No newline at end of file
+# Sirius Pipeline
+
+## Build the environment
+```
+make docker base
+make bm
```

Comment:
This fails for me:
```
sirius-pipeline$ make bm
docker run -it --name simple_switch -v /home/chris/nvidia/DASH-1/sirius-pipeline/bmv2:/bmv2 -v /home/chris/nvidia/DASH-1/sirius-pipeline/behavioral-model:/bm-src --network=host --rm bmv2-base bash -c 'cd /bm-src && ./autogen.sh && ./configure && make'
bash: ./autogen.sh: No such file or directory
Makefile:36: recipe for target 'bm' failed
make: *** [bm] Error 127
```
When I look inside the ./behavioral-model directory which is volume-mounted to `bm-src`, I see it is empty. I also noticed the Dockerfile.base has this code:
```
RUN git clone https://github.com/marian-pritsak/behavioral-model && \
        cd behavioral-model && \
	bash install_deps.sh && \
        cd .. && \
        rm -rf behavioral-model
```
The last line appears to delete you cloned, was this intentional?

Reply:
I encountered the same issue -- "make bm" fails.  Maybe some files are missing from the PR?

Reply:
bm stage is removed